src/Common.cpp

```diff
@@ -0,0 +1,47 @@
+#include "clang/AST/ASTConsumer.h"
+#include "clang/Frontend/CompilerInstance.h"
+#include "clang/Tooling/CommonOptionsParser.h"
+#include "llvm/Support/raw_ostream.h"
+
+#include "../include/BaseTransformation.h"
+#include "../include/Common.h"
+
+using clang::ASTFrontendAction, clang::ASTConsumer, clang::CompilerInstance,
+    clang::StringRef, clang::Rewriter, clang::RecursiveASTVisitor,
+    clang::DeclGroupRef, clang::SourceManager, clang::ASTContext,
+    clang::ASTMutationListener, clang::ASTDeserializationListener;
+using llvm::outs;
+using std::unique_ptr, std::vector, std::move;
+
+// ----------- Frontend Actions ------------ //
+
+BaseFrontendAction::BaseFrontendAction(
+    vector<BaseTransformation> transformations, string output_path):
+        ASTFrontendAction(),
+        transformations(move(transformations)),
+        output_path(move(output_path)) {}
+
+unique_ptr<ASTConsumer> BaseFrontendAction::CreateASTConsumer(
+        CompilerInstance &CI, StringRef file) {
```

Comment:
I don't see anything about it in a style guide. But it will be more accurate if you put a close bracket to the next line.